src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,30 @@
+package duke;
+
+
```

Comment:
It would be better if you kept the spaces between your package and imports to just 1 white line instead of 2.